about_processes.exs

```diff
@@ -0,0 +1,37 @@
+defmodule AboutProcesses do
+  use Koans
+
+  think "You are a process with a process ID" do
+    pid_name = inspect(self)
+
+    assert_? pid_name =~ ~r/#PID<\d+\.\d+\.\d+>/
```

Comment:
If you have a koan on regular expressions before (didn't check) you could do something like this:
```
    pid_format = Regex.compile!(__?)
   assert pid_name =~ format
```